oktoberfest/preprocessing/preprocessing.py

```diff
@@ -133,7 +135,9 @@ def digest(
     )
 
 
-def filter_peptides_for_model(peptides: pd.DataFrame, model: str) -> pd.DataFrame:
+def filter_peptides_for_model(
+    peptides: Optional[Union[pd.DataFrame, AnnData]], model: str
+) -> Optional[Union[pd.DataFrame, AnnData]]:
```

Comment:
this is wrong, I think. The peptides is always a dataframe afaik! Also the return type is not optional, because the function can never return None!

Reply:
I checked it and on this branch it is an AnnData object. I will try to find what change caused that. Do you think I should just remove the pd dataframe instance and only accept an AnnData object?